spec.html

```diff
@@ -38154,7 +38177,9 @@ <h1>JSON.parse ( _text_ [ , _reviver_ ] )</h1>
         1. Let _jsonString_ be ? ToString(_text_).
         1. Parse ! UTF16DecodeString(_jsonString_) as a JSON text as specified in ECMA-404. Throw a *SyntaxError* exception if it is not a valid JSON text as defined in that specification.
         1. Let _scriptString_ be the string-concatenation of *"("*, _jsonString_, and *");"*.
-        1. Let _completion_ be the result of parsing and evaluating ! UTF16DecodeString(_scriptString_) as if it was the source text of an ECMAScript |Script|. The extended PropertyDefinitionEvaluation semantics defined in <emu-xref href="#sec-__proto__-property-names-in-object-initializers"></emu-xref> must not be used during the evaluation.
+        1. Let _script_ be ParseText(! UTF16DecodeString(_scriptString_), |Script|).
```

Comment:
```suggestion
        1. Let _script_ be ParseText(! UTF16DecodeString(_scriptString_), |Script|). The Early Error defined in <emu-xref href="#sec-__proto__-property-names-in-object-initializers"></emu-xref> must not be applied during the parse.
```

The original text also fails to mention that that early error mustn't be applied, which is a bug, but we should fix it here.

(The extended PropertyDefinitionEvaluation semantics _also_ need not to apply, which is covered, but the error needs its own mention, not just the evaluation semantics.)

Reply:
Does the bugfix qualify as editorial?

Reply:
Hmmm, that's tricky. I guess we should leave this change out for now and the editors will make the change as a normative commit (but probably without asking for consensus) once this PR lands.